presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -298,6 +300,21 @@ public void testJsonToArray()
                         asMap(ImmutableList.of("h1", "h2"), asList(null, null)),
                         null));
 
+        assertFunction("CAST(JSON '[" +
+                        "[1, \"two\"], " +
+                        "[3, null], " +
+                        "{\"k1\": 1, \"k2\": \"two\"}, " +
+                        "{\"k1\": 3, \"k2\": null}, " +
```

Comment:
I understand you have more extensive tests down below, but I would change this line to
`"{\"k2\": null, \"k1\": 3}, " +` nevertheless.